gadget-container/gadgettracermanager/main.go

```diff
@@ -171,6 +183,29 @@ func main() {
 		os.Exit(0)
 	}
 
+	if liveness {
+		resp, err := healthpb.NewHealthClient(conn).Check(ctx, &healthpb.HealthCheckRequest{Service: ""})
+		if err != nil {
+			stat := status.Convert(err)
+
+			if stat.Code() == codes.DeadlineExceeded {
+				fmt.Printf("Health RPC did not complete within timeout: %v", clientTimeout)
+			} else {
+				fmt.Printf("Health RPC failed: '%s'", stat.Message())
+			}
+
+			os.Exit(1)
+		}
+
+		if resp.GetStatus() != healthpb.HealthCheckResponse_SERVING {
+			fmt.Printf("Service unhealthy: %s", resp.GetStatus().String())
+			os.Exit(1)
+		}
+
+		fmt.Printf("Service healthy: %s", resp.GetStatus().String())
```

Comment:
I am wondering if we add another health check for other processes such as traceloop, then it's better to clarify which service is healthy.

```suggestion
		fmt.Printf("Gadget Tracer Manager healthy: %s", resp.GetStatus().String())
```

Reply:
Agree, I added it to the logs.